From cff568ca9ac94a0564f0751090fde3cde41a3f11 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=A9bastien=20Wilmet?= <swilmet@gnome.org>
Date: Sun, 12 Jul 2015 11:16:04 +0200
Subject: [PATCH 19/26] build: fix generation of desktop files for external
 tools

The .desktop files were not generated since there were two rules for the
same targets. So the last rule was executed, which is the mkdir command.

To be sure that the mkdir command is executed before any target is
built, we can use $(shell ...). See:
http://stackoverflow.com/questions/2122602/force-makefile-to-execute-script-before-building-targets
---
 plugins/externaltools/data/Makefile.am | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/plugins/externaltools/data/Makefile.am b/plugins/externaltools/data/Makefile.am
index e3a4707..23098be 100644
--- a/plugins/externaltools/data/Makefile.am
+++ b/plugins/externaltools/data/Makefile.am
@@ -59,5 +59,4 @@ CLEANFILES +=					\
 # generating the .desktop files.
 # If the following bug is fixed, this hack can normally be removed:
 # https://bugs.launchpad.net/intltool/+bug/605826
-$(externaltools_all_desktop_files):
-	$(AM_V_GEN) $(MKDIR_P) $(top_builddir)/plugins/externaltools/data/
+$(shell $(MKDIR_P) $(top_builddir)/plugins/externaltools/data/)
-- 
1.8.3.1

